CMakeLists.txt

```diff
@@ -1000,6 +1000,8 @@ if(WIN32)
   target_link_libraries(mixxx-lib PUBLIC shell32)
 
   if(MSVC)
+    # needed for clcache
+    target_compile_options(mixxx-lib PUBLIC /c)
```

Comment:
No idea what `/c` does? Does it have any bad side effects? Should we only add it if `CLCACHE_SUPPORT` is `ON`?